staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/types.go

```diff
@@ -16,8 +16,8 @@ limitations under the License.
 
 package encryptionconfig
 
-// EncryptionConfig stores the complete configuration for encryption providers.
-type EncryptionConfig struct {
+// Config stores the complete configuration for encryption providers.
+type Config struct {
```

Comment:
don't change the go type, this is mirroring the allowed Kind in https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go#L77